src/main/java/com/microsoft/jenkins/containeragents/aci/AciCloud.java

```diff
@@ -62,6 +70,11 @@ public AciCloud(String name,
         this.templates = templates;
     }
 
+    @DataBoundSetter
+    public void setLogAnalyticsCredentialsId(String logAnalyticsCredentialsId) {
```

Comment:
apologies I missed this in the first review, this needs some form of documentation, 

A `help-logAnalyticsCredentialsId.html` file in the 
https://github.com/jenkinsci/azure-container-agents-plugin/tree/master/src/main/resources/com/microsoft/jenkins/containeragents/aci/AciCloud directory should do it.

you've used a username password credential type,

I'm guessing the username is the workspace name, and a workspace key is the 'password'?

should be documented / linked to another document for how to get the workspace key, as I don't remember it being the most obvious

Reply:
I see username is workspace id https://github.com/jenkinsci/azure-container-agents-plugin/pull/76/files#diff-c1de754120dbcb351ce78f5fb4854f9c5ecbcf73eec5489923de3a6ec310e7e2R206-R209